bin/asini.js

```diff
@@ -39,6 +40,8 @@ var cli = meow([
   "  --repo-version       Specify repo version to publish",
   "  --concurrency        How many threads to use if asini parallelises the tasks (defaults to 4)",
   "  --loglevel           What level of logs to report (defaults to \"info\").  On failure, all logs are written to asini-debug.log in the current working directory.",
+  "  --dry                Don't eject any packages, only log where they would be moved",
```

Comment:
Similar to https://github.com/lerna/lerna/issues/51, I think we should do this as a general flag (named `--dry-run`, to be consistent with npm and git and many other CLI interfaces), not specific to the `eject` subcommand.

This complicates testing, of course...

(I'm not a fan of meow's pattern of dumping all the flags for all commands in one place, but that's a separate issue)